builtin/rebase--interactive2.c

```diff
@@ -131,57 +139,83 @@ int cmd_rebase__interactive(int argc, const char **argv, const char *prefix)
 		warning(_("--[no-]rebase-cousins has no effect without "
 			  "--rebase-merges"));
 
-	ret = prepare_branch_to_be_rebased(&opts, switch_to);
-	if (ret)
-		return !!ret;
+	switch (command) {
+	case SKIP: {
+		struct string_list merge_rr = STRING_LIST_INIT_DUP;
 
-	ret = get_revision_ranges(upstream, onto, &head_hash,
-				  &revisions, &shortrevisions);
-	if (ret)
-		return !!ret;
+		rerere_clear(&merge_rr);
+	case CONTINUE:
+		ret = sequencer_continue(&opts);
+		break;
+	}
```

Comment:
I think this brace should be after the call to rerere_clear() above.

Reply:
Also maybe a `break;` is missing after the call to rerere_clear(), otherwise maybe a `/* fall through */`comment should be added after the call to rerere_clear().

Reply:
I am not sure about the style. Let's leave it like this and see what people on the list say.